client/layout/masterbar/style.scss

```diff
@@ -288,26 +288,40 @@ body.is-mobile-app-view {
 	justify-content: center;
 
 	&.masterbar__reader {
-		padding: 0 7px 0 4px;
+		padding: 0 10px 0 6px;
 
 		.masterbar_svg-reader {
-			padding: 0 8px 0 3px;
+			padding: 0 0 0 4px;
 		}
 	}
 
+	&.masterbar__item-my-site,
 	&.masterbar__item-my-site-actions {
 		padding-left: 7px;
 	}
 
 	@media only screen and (max-width: 781px) {
-		&.masterbar__reader,
+		&.masterbar__item-my-site {
+			padding-left: inherit;
+		}
 		&.masterbar__item-my-site-actions {
 			padding: 0;
 		}
 
 		&.masterbar__reader {
+			padding: 0 8px;
+
 			.masterbar_svg-reader {
-				padding: 0;
+				padding: 0 0 0 2px;
+			}
+
+			&.is-active {
+				padding: 0 2px 0 0;
+				margin: 0 6px 0 4px;
+
+				.masterbar_svg-reader {
+					padding: 0 0 0 2px;
+				}
```

Comment:
It looks like this padding is already applied just above to the same class selector as well (outside of is-active)

Reply:
Removed!